common/xbps-src/shutils/common.sh

```diff
@@ -597,4 +597,14 @@ setup_pkg() {
     fi
 
     source_file $XBPS_COMMONDIR/environment/build-style/${build_style}.sh
+
+    # Source all build-env files that are defined
+    if [ -n "$build_env" ]; then
+        for f in $build_env; do
```

Comment:
I think you can omit the -n test above this and the outcome will be the same (i.e., no loop iterations), similar to the changes to show.sh

Reply:
I know but everything else does it so i'm doing as well

Reply:
Makes sense to me -- there might be a reason for it I'm just not aware of.